webextensions/api/userScripts_legacy.json

```diff
@@ -0,0 +1,126 @@
+{
+  "webextensions": {
+    "api": {
+      "userScripts_legacy": {
+        "__compat": {
+          "description": "userScripts (Legacy)",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "68",
+              "notes": "Only available for use in extensions using Manifest V2."
+            },
+            "firefox_android": "mirror",
+            "opera": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror"
+          }
+        },
+        "RegisteredUserScript": {
+          "__compat": {
+            "description": "RegisteredUserScript (Legacy)",
```

Comment:
@rebloor, I'm curious about your motivations for including "(Legacy)" on each of the entries in this data. Is this an established practice? Does it affect how members appear in compatibility tables or in other tooling?

Reply:
As far as I'm aware, we haven't had a similar situation in the past, so we're inventing "practice" here. In this situation, we have a `userScript` API implemented in Firefox only for Manifest V2 and an incompatible `userScript` API for Manifest V3 implemented in Firefox, Chrome, and Safari. In the MDN documentation readers will see the Firefox only version of the API marked up as "legacy" throughout. See https://github.com/mdn/content/pull/37975  `description` is used as the title in the compatibility tables.

Reply:
I think I may have suggested this. If we render the BCD table for this feature, there would be no indication otherwise that this refers to the old API.

I notice that the features have no status set (like experimental, deprecated), which presumably is a special treatment. If it is forbidden to mark a webextensions.api feature as deprecated, then we should probably change that.

Reply:
Unless there's a change I'm unaware of, we have no plans to deprecate Manifest V2 ((https://blog.mozilla.org/addons/2024/03/13/manifest-v3-manifest-v2-march-2024-update/) so by extension we presumably have no plans to deprecate the Manifest V2 version of `userScript`. @Rob--W 

In addition to including the word legacy in the descriptions, wouldn't the `mdn_url` indicate that it refers to the legacy API?